lib/ma_crossover/events/life_start.js

```diff
@@ -19,11 +19,14 @@ const onLifeStart = async (instance = {}) => {
 
   debug(
     'starting with long %s(%d) and short %s(%d)',
-    long.type.toUpperCase, long.args[0], short.type.toUpperCase(), short.args[0]
+    long.type.toUpperCase(), long.args[0], short.type.toUpperCase(), short.args[0]
   )
 
-  const LongIndicatorClass = HFI[long.type.toUpperCase()]
-  const ShortIndicatorClass = HFI[short.type.toUpperCase()]
+  const typeL = long.type.toUpperCase() === 'MA' ? 'SMA' : long.type.toUpperCase()
```

Comment:
`'MA' ? 'SMA'` -> `'MA' ? 'LMA'`? I might be missing something though

Reply:
Simple mean average
Exponential mean average

I checked for sure that MA/SMA has the "wrong naming", but other types i don't want to touch.